docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -45,6 +45,29 @@ There you can find links to upgrade notes for other versions too.
     - from `getPaginatedProductDetailsInCategory` to `getPaginatedProductsInCategory`
     - from `getPaginatedProductDetailsForBrand` to `getPaginatedProductsForBrand`
     - from `getPaginatedProductDetailsForSearch` to `getPaginatedProductsForSearch`
+- remove usages of inherited `OrderItem` classes ([#715](https://github.com/shopsys/shopsys/pull/715))
+    - replace usages of `OrderProduct`, `OrderPayment`, and `OrderTransport` with common `OrderItem`
+    - replace usages of `OrderTransportData`, `OrderPaymentData` with common `OrderItemData`
+    - replace usages of `OrderProductFactoryInterface` with common `OrderItemFactoryInterface`
+        - replace usages of `OrderProductFactory` with `OrderItemFactory`
+        - replace usages of method `create()` with `createProduct()`
+    - replace usages of `OrderPaymentFactoryInterface` with common `OrderItemFactoryInterface`
+        - replace usages of `OrderPaymentFactory` with `OrderItemFactory`
+        - replace usages of method `create()` with `createPayment()`
+    - replace usages of `OrderTransportFactoryInterface` with common `OrderItemFactoryInterface`
+        - replace usages of `OrderTransportFactory` with `OrderItemFactory`
+        - replace usages of method `create()` with `createTransport()`
+    - replace usages of `OrderPaymentDataFactoryInterface` with common `OrderItemDataFactoryInterface`
+        - replace usages of `OrderPaymentDataFactory` with common `OrderItemDataFactory`
+        - replaces usages of method `createFromOrderPayment()` with `createFromOrderItem()`
+    - replace usages of `OrderTransportDataFactoryInterface` with common `OrderItemDataFactoryInterface`
+        - replace usages of `OrderTransportDataFactory` with common `OrderItemDataFactory`
+        - replaces usages of method `createFromOrderTransport()` with `createFromOrderItem()`
+    - following classes changed constructors, if you extend them, change them appropriately:
+        - `Order`
+        - `OrderDataFactory`
+        - `OrderItemFacade`
+        - `OrderFacade`
```

Comment:
I can think of two neccessary steps that are not included in the upgrade notes:
- using the `isType<type>()` method instead of `instanceof`
- removal of the non-existing test cases from `EntityExtensionTest.php`